src/client/server.js

```diff
@@ -1,8 +1,20 @@
 const express = require('express');
 const app = express();
 
-app.use(express.static('static')); // serve the user the ./static folder with all the .html and .js files for local rendering
+app.use(express.static('.')); // serve the user the ./static folder with all the .html and .js files for local rendering
```

Comment:
Folder _node_modules_ has to be accessible for monaco editor, therefore root cannot be in a sub-folder.